var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -173,7 +176,7 @@ class PyTorch(PythonPackage, CudaPackage):
     # https://github.com/pytorch/pytorch/issues/60328
     patch('https://github.com/pytorch/pytorch/pull/59220.patch?full_index=1',
           sha256='6d5717267f901e8ee493dfacd08734d9bcc48ad29a76ca9ef702368e96bee675',
-          when='@1.2:')
+          when='@1.2:1.11')
```

Comment:
Patch no longer applies to master, and should be merged any day now anyway: https://github.com/pytorch/pytorch/pull/78037